scripts/kallisto2ga.py

```diff
@@ -0,0 +1,135 @@
+"""
+    Script to parse the abundances.txt output file produced by kallisto.  As
+    current version of the rnaQuant is using a flat file for the data this
+    parser is living in server/scripts.  Once the backend store is ready for
+    rnaQuant then this functionality will move to server/loaders.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+import errno
+
+
+def makeDir(path):
```

Comment:
if you find yourself using the same methods more than once, consider putting them in `scripts/utils.py`